doc/password-hashing-functions/index.md

```diff
@@ -0,0 +1,97 @@
+# Password hashing fucntions
+
+!!! info
+
+    **Status**: WIP RF
```

Comment:
Status: [Proposed](https://gig-cymru-nhs-wales.github.io/Architecture-Decision-Records/design-authority/dhcw/architecture-decision-record-process/#lifecycle-and-status)